test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml

```diff
@@ -0,0 +1,7 @@
+let () =
+  let _ = Sys.command "touch file" in
```

Comment:
Why is creating this file needed?

Reply:
Can't the bug be reproduced with a broken symlink?

Reply:
I don't know how to make a broken one.